x-pack/plugins/infra/public/pages/logs/page_content.tsx

```diff
@@ -7,6 +7,7 @@
 import { i18n } from '@kbn/i18n';
 import React from 'react';
 import { Route, Switch } from 'react-router-dom';
+import { useMount } from 'react-use';
```

Comment:
Oh nice. Didn’t know about this. I feel like I end up saying that whenever you use things from this library 😅 I can use this in my alerting work 👍

Reply:
Indeed, it has so many useful hooks which we could use to replace our own, slightly hacky implementations. And even though they are mostly pretty simple their availability just removes many papercuts (e.g. `useToggle` vs  yet another custom `useState<boolean>()` with several `const on = useCallback(...)`).